### PR TITLE
Fix form, add logic

### PR DESCRIPTION
**Pull request by quangkeu95**

### CODE REVIEW

File: src/js/containers/LimitOrder/LimitOrder.js [hunk 1]
```diff
@@ -67,6 +68,15 @@ export default class LimitOrder extends React.Component {
     clearInterval(this.invervalProcess)
   }
 
+  async getOrders() {
+    try {
+      const results = await limitOrderServices.getOrders();
```
Comment (andrewsource147) [inline src/js/containers/LimitOrder/LimitOrder.js]: getOrders only call when user has login, should check user login

File: src/js/containers/LimitOrder/LimitOrderList.js [hunk 2]
```diff
@@ -142,11 +30,13 @@ export default class LimitOrderList extends React.Component {
       },
       pairFilterVisible: false,
       statusFilterVisible: false,
+      cancelOrderModalVisible: false,
```
Comment (andrewsource147) [inline src/js/containers/LimitOrder/LimitOrderList.js]: Why cancelOrderModalVisible is in global state. Can it be in local state?

File: src/js/reducers/limitOrderReducer.js [hunk 3]
```diff
@@ -146,14 +146,44 @@ const limitOrder = (state = initState, action) => {
...
+      const { order } = action.payload;
+      const listOrder = JSON.parse(JSON.stringify(newState.listOrder));
+      const target = listOrder.filter(item => item.id === order.id);
+      if (target.length > 0) {
+        listOrder.splice(listOrder.indexOf(target[0]), 1);
+      }
+      newState.listOrder = listOrder;
+      return newState;
+    }
+    case "LIMIT_ORDER.CANCEL_ORDER": {
```
Comment (andrewsource147) [inline src/js/reducers/limitOrderReducer.js]: Its should call to limit order services got cancel order. You should not hard code for status "cancel"

Comment (quangkeu95) [reply src/js/reducers/limitOrderReducer.js]: Got it, thanks for pointing out.

File: src/js/containers/LimitOrder/LimitOrder.js (hunk 1 above)
Comment (quangkeu95) [reply src/js/containers/LimitOrder/LimitOrder.js]: Got it, thanks!

**Pull request merged**